src/lib/deploy-actions.js

```diff
@@ -23,8 +23,8 @@ const { deployActions } = require('@adobe/aio-lib-runtime')
  */
 /** @private */
 module.exports = async (config, isLocalDev = false, log = () => {}, filter = false, inprocHook) => {
-  utils.runScript(config.hooks['pre-app-deploy'])
-  const script = await utils.runScript(config.hooks['deploy-actions'])
+  runInProcess(config.hooks['pre-app-deploy'], config)
```

Comment:
similar comment like I had regarding `pre-app-build` above (for pre-app-deploy and post-app-deploy)

Reply:
Done